perf/perf_c2c.py

```diff
@@ -88,20 +89,41 @@ def run_cmd(self, cmd):
         except process.CmdError as details:
             self.fail("Command %s failed: %s" % (cmd, details))
 
+    def run_record_cmd(self, cmd):
+        """
+        Run record command
+        """
+        try:
+            cmd_output = subprocess.Popen(cmd,
```

Comment:
ah why not use `process` API from avocado,as I know this is a data drainer issue in avocado process API , can  you please have look